api/src/main/java/org/pmiops/workbench/api/StatusAlertController.java

```diff
@@ -22,25 +25,24 @@ public class StatusAlertController implements StatusAlertApiDelegate {
   }
 
   @Override
-  public ResponseEntity<StatusAlert> getStatusAlert() {
+  public ResponseEntity<List<StatusAlert>> getStatusAlerts() {
     return ResponseEntity.ok(
         statusAlertDao
-            .findFirstByOrderByStatusAlertIdDesc()
+            .findAllByOrderByStatusAlertIdDesc()
+            .stream()
             .map(statusAlertMapper::toStatusAlert)
-            .orElse(new StatusAlert()));
+            .collect(Collectors.toList()));
+  }
+
+  @Override
+  public ResponseEntity<EmptyResponse> deleteStatusAlert(Long id) {
```

Comment:
The UI used to toggle alerts by only keeping a single status alert and deleting old alerts. The creating and deleting were in a single endpoint.

This creates a separate endpoint for deleting alerts.